capture_gui/lib.py

```diff
@@ -290,6 +289,17 @@ def default_output():
     return os.path.join(workspace, folder, filename)
 
 
+def get_project_rule(rule):
+    """Get the full path of the rule of the project"""
+    workspace = cmds.workspace(query=True, rootDirectory=True)
+    folder = cmds.workspace(fileRuleEntry=rule)
+    if not folder:
+        log.warning("File Rule Entry '{}' has no value, please check if the "
```

Comment:
I'm not too sure about this; in the AbcExport plug-in they had the same behavior: https://github.com/alembic/alembic/blob/master/maya/AbcExport/AbcExport.cpp#L606

If no non-empty value was set in the file rule then it was considered to *not* be set at all. Though on the other hand, it wouldn't log verbosely, it would just fix it itself towards a default.

Let's stick with the logging as it is and see if it ever introduces confusion.